### PR TITLE
Code cleanup per TSLint warnings

### PR DESCRIPTION
**Pull request by mfcallahan**
Some small changes to fix TSLint warnings and improve code quality/consistency per this project's "tslint.json" file:

- Use single quotes (Rule: quotemark)
- Add missing semicolons (Rule: semicolon)
- End file "app.component.ts" with new line (Rule: eofline)
- Remove unnecessary uses of `tslint:disable-next-line`
- Remove unused arrow function parameter

### CODE REVIEW

File: esm-samples/jsapi-angular-cli/src/app/app.component.ts [hunk 1]
```diff
@@ -70,31 +70,29 @@ export class AppComponent implements OnInit, OnDestroy {
     return this.view.when();
   }
 
-  // tslint:disable-next-line:use-lifecycle-interface
```
Comment (mfcallahan) [inline esm-samples/jsapi-angular-cli/src/app/app.component.ts:73]: Class `AppComponent` implements interface `OnInit`, this comment does not have any effect.

File: esm-samples/jsapi-angular-cli/src/app/app.component.ts [hunk 2]
```diff
@@ -70,31 +70,29 @@ export class AppComponent implements OnInit, OnDestroy {
...
         this.zone.run(() => {
           console.log('mapView ready: ');
-        })
+        });
       });
 
     });
   }
 
-  // tslint:disable-next-line:use-lifecycle-interface
```
Comment (mfcallahan) [inline esm-samples/jsapi-angular-cli/src/app/app.component.ts:93]: Class `AppComponent` implements interface `OnDestroy` - removed this comment as it does not have any effect.

**Comment by mfcallahan** [comment]
I was experimenting with the new ArcGIS JS API new ES modules in Angular and per Esri's contributing guidelines, I thought I'd open a quick PR to fix a few things I noticed. Great work so far, I'm really excited to be testing the new ES Modules. Everything is working great so far and I can't wait until this is production ready!!

**Comment by andygup** [comment]
Thanks Matt!! Merging and closing.

**Pull request merged**